javascript/src/fizzbuzz.js

```diff
@@ -5,6 +5,8 @@
     if(input % 5 === 0) 
         output += 'Buzz'
     if(input % 99 === 0) output += 'Strange'
+    if(input % 101 === 0) output += 'Even Stranger'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/dan-codacy/monorepo/pullRequest?prid=8124379)
```suggestion
    if(input % 101 === 0) output += "Even Stranger"
```